EmpleoDotNet/App_Start/EmpleadoModule.cs

```diff
@@ -16,8 +19,12 @@ public override void Load()
 
             Kernel.Bind<IJobOpportunityRepository>().To<JobOpportunityRepository>();
             Kernel.Bind<ITagRepository>().To<TagRepository>();
+            Kernel.Bind<IUserProfileRepository>().To<UserProfileRepository>();
 
             Kernel.Bind<IJobOpportunityService>().To<JobOpportunityService>();
+            Kernel.Bind<IAuthenticationService>().To<AuthenticationService>()
+                .WithConstructorArgument("userManager", new UserManager<IdentityUser>(new UserStore<IdentityUser>(new EmpleadoContext())));
```

Comment:
Me gustaría ver si podemos hacerle tests al authentication service. Idealmente fuera en este mismo PR, pero todavía no tenemos como estándar obligar que los features nuevos tengan unit tests, así que eso sería a opción de ustedes. Eso sí, puede que se haga un poco difícil con este objeto usermanager

Entonces sería bueno identificar:
1. Los métodos que usamos del user manager están marcados como `virtual`? Si ese es el caso solo tenemos que pasar un Substitute.For<UserManager> en el test del authentication service.
2. De otro modo, pudiéramos poner este usermanager en un wrapper class (pudiera estar en el repo layer ya que está bregando directamente con el context object) para que sea fácil hacer stubs de ese wrapper?